framework/src/media/audio/audio_manager.c

```diff
@@ -187,7 +189,17 @@ static audio_manager_result_t find_audio_card(audio_card_type_t card_type)
 		}
 
 		if (type == type_chr) {
+			if (card[card_id].status > 0) {
```

Comment:
It does not necessary, we call find_audio_card if g_actual_audio_in/out_card_id is invalid, also g_actual_audio_in/out_card_id set inside of get_actual_audio_in/out_card_id where already checks status.

Reply:
You're right. I'll delete.
@jsdosa It is found that this find_audio_card() is executed only once when g_actual_audio_in/out_card_id is invalid as TJ's comment indicates. Therefore pthread_mutex_init() is also called only once.

Reply:
In my opinion, it seems that you need to test by creating two threads. This is because mutex_init can be done again.

Reply:
@jsdosa Oops, that was your point, right...
@Taejun-Kwon to avoid the tho threads situation, I think we need 'audio_manager_init()' or it is difficult to resolve this case. What do you think..?